specification/dimensions/billingaccountid.md

```diff
@@ -0,0 +1,32 @@
+# Billing Account ID
+
+A billing account is a container for resources and/or services that are billed together in an invoice. Common scenarios for using billing accounts include grouping based on organizational constructs and hierarchy-based cost allocation approach.
+
+A billing account ID is a provider assigned identifier for a billing account. It is commonly used for invoice reconciliation and cost allocation scenarios.
```

Comment:
I suggest that we combine these two usage-related sentences ("_Common scenarios..._" and "_It is commonly used..._") into one. 
We could add the _invoice reconciliation_ reference to the first one and leave out the second?

Note: The same applies to the **billing account name spec** and corresponding **appendix section**.

Reply:
Just had this discussion with Michael Flanakin as well. Technically, the first paragraph is for the billing account and the second is for billing account id. I think maybe its best to do what you're saying here and merge to the top paragraph and then people can select if they want to use the billing account id or name to do these things.

Reply:
As @rileyjenk mentioned, unlike billing account ID, billing account name (i.e., name dimensions in general) are subject to change and thus we would prefer using IDs for many of these UC scenarios (cost allocation in particular). 
We might consider including this as a constraint or at least in the supporting content?

Reply:
As long as we say it may change I think it is ok and does improve immediate human readability. Over time is where we need to specify that it can be subject to change.

Reply:
Resolved by Udam's change